test/compeon/rack_tools/pipes/parse_oauth2_authorization_code_test.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+require 'compeon/rack_tools/pipes/parse_oauth2_authorization_code'
+
+module Compeon
+  module RackTools
+    module Pipes
+      class ParseOauth2AuthorizationCodeTest < Minitest::Test
+        def test_with_a_valid_code
+          api_token_stub = :api_token_stub
+
+          Compeon::AccessToken.stub :parse, api_token_stub do
```

Comment:
Why do you stub here? You can use a real token here. If you stab here you can't be sure that the arguments are right.
UPDATE: leave it as it is, we can fix it if we have mocking support in `Compeon::AccessToken`

Reply:
That which arguments are right?

Reply:
the argument which you pass into `parse`

Reply:
You can already mock `Compeon::AccessToken` by the way:
```ruby
KEY = OpenSSL::PKey::RSA.new(512)

def setup
  Compeon::AccessToken.environment = 'test'
  Compeon::AccessToken.public_key_string = KEY.public_key.to_s
end
```